src/Sylius/Bundle/InventoryBundle/Resources/config/doctrine/model/InventoryUnit.orm.xml

```diff
@@ -24,7 +24,7 @@
 
         <field name="inventoryState" column="inventory_state" type="string" />
 
-        <many-to-one field="stockable" target-entity="Sylius\Component\Inventory\Model\StockableInterface">
+        <many-to-one field="stockable" target-entity="Sylius\Component\Core\Model\ProductVariant">
```

Comment:
Sure it should not! I'm aware about some decoupling issues that can appear in this PR and I will be grateful for pointing them out - it will make my work much easier ;)